Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -518,6 +518,28 @@ class MultiTrajectory {
     return self().template measurementCovariance_impl<measdim>(covIdx);
   }
 
+  template <bool RO = ReadOnly, typename = std::enable_if_t<!RO>>
+  typename TrackStateProxy::EffectiveMeasurement effectiveMeasurement(
+      IndexType measIdx) {
```

Comment:
Re `measIdx`: these are the state indices, right? There is no dedicated measurement index, like there is a dedicated track covariance index.